### ISSUE TITLE
Clarity & Monaco Editor in a Form

### ISSUE THREAD

**Issue by windischb** [vmware/clarity#1580]
```
[x] bug
[ ] feature request
[ ] enhancement
```

### Expected behavior
Set Cursor to last position on line, if click on an empty Area in line.

### Actual behavior
Nothing happend when click on a line...
Works correct in Firefox!
The Problem only exists on Chrome.

### Reproduction of behavior
Working example without clarity -> https://jsfiddle.net/c4Lb9ybb/
Not Working exampe with clarity -> https://jsfiddle.net/9vqz2x0c/
It does also works in Chrome if i remove the <form>

### Environment details

* **Clarity version: 0.10.0**

* **OS and version:** 

* **Browser:** Chrome 61.0.3163.100

**Comment by adityarb88** [comment]
@windischb: I believe this issue is because inside of Clarity forms, we target the `span` selector directly. You can do this in your app CSS for now:

```
.view-line span {
 display: block
}
```

to resolve the issue. But this will be resolved with #932.

Closing the issue based on the above info. Please feel free to reopen the issue if you have any more questions.

**Comment by windischb** [comment]
You are right!

```css
.view-line span {
    display: inline
}
```

solved my problem.

Many thanks!
